scripts/telemetry/README.md

```diff
@@ -0,0 +1,41 @@
+UFM Telemetry
+--------------------------------------------------------
+
+
+This plugin is used to load UFM telemetry counters and metrics.
+
+Overview
+--------------------------------------------------------
+
+NVIDIA UFM Telemetry platform provides network validation tools to monitor network performance and conditions, capturing and streaming rich real-time network telemetry information, application workload usage to an on-premise or cloud-based database for further analysis.
+As a fabric manager, the UFM Telemetry holds a real-time network telemetry information of the network topology.
+
+
+Prerequisites
+--------------------------------------------------------
+
+To install all prerequisites, please run :
+
+    pip install -r requirements.txt
+
+To exported your repository to PYTHONPATH, please run :
+
+   export PYTHONPATH="${PYTHONPATH}:<your ufm_sdk_cookbook path>"
+
+
+Run
+--------------------------------------------------------
+### 1. Get all events :
+
+    python3 load_telemetry.py --ufm_telemetry_host=<ufm_telemetry_host> --ufm_telemetry_port=<ufm_telemetry_port> --ufm_telemetry_url=<ufm_telemetry_url>
```

Comment:
please explain in the readme that we have two modes to pass the configurations, the conf file and the inline